lib/arboreal/class_methods.rb

```diff
@@ -10,6 +10,7 @@ def rebuild_ancestry
       begin
         n_changes = extend_materialized_paths
       end until n_changes.zero?
+      populate_root_relation if reflect_on_association(:root_ancestor)
```

Comment:
It's supported in Rails 3 & 4 `http://apidock.com/rails/ActiveRecord/Reflection/ClassMethods/reflect_on_association`